psi4/src/read_options.cc

```diff
@@ -247,6 +247,20 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     /*- How many NOONS to print -- used in libscf_solver/uhf.cc and libmints/oeprop.cc -*/
     options.add_str("PRINT_NOONS", "3");
 
+    ///MBIS Options (libmints/oeprop.cc)
+
+    /*- Maximum Number of MBIS Iterations -*/
+    options.add_int("MBIS_MAXITER", 500);
+    /*- MBIS Convergence Criteria -*/
+    options.add_double("MBIS_D_CONVERGENCE", 1.0e-8);
+    /*- MBIS Number of Radial Points -*/
+    /*- Additional Radial and/or Spherical Points may be needed for Heavier Atoms (200-300) like Zinc -*/
+    options.add_int("MBIS_RADIAL_POINTS", 75);
+    /*- MBIS Number of Spherical Points -*/
+    options.add_int("MBIS_SPHERICAL_POINTS", 302);
+    /*- Pruning scheme for MBIS Grid -*/
+    options.add_str("MBIS_PRUNING_SCHEME", "NONE");
```

Comment:
* you're going to want this option to look more like the `DFT_PRUNING_SCHEME` entry, particularly by giving it a third argument for allowed values so it functions like an enum. a string w/o third arg, like at present, can take any value, so `set mbis_pruning_scheme bulldog` allowed.
* may as well make the default value (second arg) `ROBUST` as Holger suggests

Reply:
@hokru I ran `mbis_pruning_scheme robust` on ZnO, and I got an egregious error in the number of electrons as calculated by the grid (37.784030), whereas the number of electrons is fine when I run with all of the same options using `mbis_pruning_scheme none` (38.00079). 

For reference, here are the other grid options

scf_type df
d_convergence 8
e_convergence 10
mbis_radial_points 300
mbis_spherical_points 350

`mbis_pruning_scheme robust` worked perfectly fine for my other 4 test systems.

Is this to be expected? If so, should I leave a comment in the ZnO test system, and put a suggestion to turn off pruning in the error message?

Reply:
hmm, that deviation looks suspiciously large actually. With PBE/def2-TZVP and standard grids i get much smaller deviations in ORCA and Turbomole (N=37.999..).

Reply:
Andy, try adding a print grid call to the mbis https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libfock/cubature.cc#L4490 to make quite sure you're getting what you ordered up.

Reply:
I got similar results for the normal DFT grid. My feeling that something is wrong was right, bug fix submitted. Zn should behave normal after that.

Reply:
After implementing the bug fix, I got a much better electron count for ZnO (error of 4E-5, with robust pruning). I updated the test for ZnO, changing the reference values to match those given after the bug fix. I didn't commit the bug fix into MBIS for this PR, as it is a separate PR. Thank you for catching the error @hokru